harness/tests/determined/common/experimental/test_determined.py

```diff
@@ -0,0 +1,71 @@
+from typing import Callable
+from unittest import mock
+
+import pytest
+import responses
+
+from determined.common.api import authentication, errors
+from determined.common.experimental import Determined
+from tests.fixtures import api_responses
+
+_MASTER = "http://localhost:8080"
+
+
+@pytest.fixture
```

Comment:
This is nice. It'd have taken me a lot of work to figure out how to get auth mocked for a `Determined()`.

Reply:
I also like the object you've chosen to give the test to.